copilot/fastapi/app/config.py

```
@@ -0,0 +1,91 @@
+    
+    # Security
+    verify_ssl: bool = os.getenv("VERIFY_SSL", "true").lower() != "false"
+    
+    class Config:
+        env_file = ".env"
+        
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self._check_security_settings()
+    
+    def _check_security_settings(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_check_security_settings`

Add return type annotation: `None`

(ANN202)